plugin/teksi_wastewater/interlis/gui/interlis_export_settings_dialog.py

```diff
@@ -106,3 +112,10 @@ def selected_model(self):
 
     def selected_models(self):
         return self.export_model_selection_comboBox.currentData()
+
+    @property
+    def selected_orientation(self):
+        # neu 23.4.2024 added float()
+        eorientation = self.export_orientation_selection_comboBox.currentData()
+        print(eorientation)
```

Comment:
Print should be removed or replaced by a log

Reply:
ok